tox.ini

```diff
@@ -0,0 +1,45 @@
+[tox]
+envlist= py38,style,bandit
+
+[testenv]
+extras= test
+# astropy will complain if the home directory is missing
+passenv= HOME
+commands=
+    pytest --open-files tests
```

Comment:
`--open-files` is already in `setup.cfg`, so probably not needed here?

Reply:
Yes good point!